_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About search telemetry
+
+You can use search telemetry to analyze performance for success or failed search requests in OpenSearch Dashboards. OpenSearch stores telemetry data in the `.kibana_1` index.
+
+Because there are thousands of concurrent search request from OpenSearch Dashboards, the large traffic causes significant load in an OpenSearch cluster.
+
+OpenSearch clusters perform better with search telemetry turned off.
+{: .tip }
+
+## Enable search telemetry
+
+Search usage telemetry is disabled by default. To enable it, you need to change the OpenSearch Dashboards YAML file `opensearch_dashboards.yml` setting `data.search.usageTelemetry.enabled` to `true`.
```

Comment:
"To enable it, you need to set `data.searchusageTelemtry.enabled` to `true` in `opensearch_dashboards.yml`.